health/health/src/main/java/io/helidon/health/HealthSupport.java

```diff
@@ -212,6 +212,7 @@ HealthResponse callHealthChecks(List<HealthCheck> healthChecks) {
                 .collect(Collectors.toList());
 
         Status status = responses.stream()
+                .filter(this::internalErrorExcluded)
```

Comment:
IMO it would be cleaner (less code added) to use
```java
.filter(Predicate.not(HcResponse::internalError))
```
here and remove the new  `internalErrorExcluded` method.